### DISCUSSION TITLE
LTE Modem on version 2022.45.1

### DISCUSSION THREAD

**Discussion by deanovadia** [tesla-android/issue-tracker#188] [Q&A]
Hi,
I have installed 2022.45.1 and it works great!
Trying to use an LTE 4G modem I have [https://a.co/d/7wNAhYA](url) 
I have configured the modem using a PC to allow connections via USB.
Not sure how to make this work.. The Android won't recognize the 4G cellular modem.
Somebody tried this?
TY!

**Comment by adamChrystie** [comment]
The link you provided is not valid. We do not know what type of modem you have. Please update the thread with the info. Have a good day.

**Reply by edwbking** [reply]
I just tried a Franklin T10 usb tethered and it works well.

**Comment by josher32** [comment]
I’m also having trouble using ALCATEL LTE model IK41UC usb stick with 45.1.  Doesn’t seem to recognize even though stick works fine on cellular in computer.  Any help would be greatly appreciated!

**Reply by adamChrystie** [reply]
Hi @josher32 someone else has this same issue and posted on the same day as you...what a coincidence..life is funny like that sometimes eh? Monitor this Reddit thread, "[Unable to get Alcatel LTE USB stick](https://www.reddit.com/r/teslaandroid/comments/zsxf6r/unable_to_get_alcatel_lte_usb_stick/?utm_source=share&utm_medium=web2x&context=3)"
